src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -6099,6 +6135,21 @@ static FlowAnalysisAnnotations getPropertyAnnotations(SourcePropertySymbol prope
             }
         }
 
+        private static FlowAnalysisAnnotations ToInwardAnnotations(FlowAnalysisAnnotations outwardAnnotations)
+        {
+            var annotations = FlowAnalysisAnnotations.None;
+            if ((outwardAnnotations & FlowAnalysisAnnotations.MaybeNull) != 0)
+            {
+                // MaybeNull and MaybeNullWhen count as MaybeNull
+                annotations |= FlowAnalysisAnnotations.AllowNull;
+            }
+            if ((outwardAnnotations & FlowAnalysisAnnotations.NotNull) == FlowAnalysisAnnotations.NotNull)
```

Comment:
Style between this and the above `if` feels inconsistent

Reply:
Yes, that's on purpose. MaybeNull is two bits, either of which should count (see comment above).
NotNull is also two bits, but we are lenient (we ignore NotNullWhenTrue and NotNullWhenFalse on their own here).

---
In reply to: [368161275](https://github.com/dotnet/roslyn/pull/40789#discussion_r368161275) [](ancestors = 368161275)

Reply:
I'll add a comment

---
In reply to: [368183067](https://github.com/dotnet/roslyn/pull/40789#discussion_r368183067) [](ancestors = 368183067,368161275)